packages/mobile/src/components/comments/CommentOverflowMenu.tsx

```diff
@@ -148,7 +148,8 @@ export const CommentOverflowMenu = (props: CommentOverflowMenuProps) => {
     },
     isCommentOwner && {
       text: messages.menuActions.edit,
-      callback: () => setEditingComment?.(props.comment)
+      callback: () =>
+        setReplyingAndEditingState?.({ editingComment: props.comment })
     },
```

Comment:
im cool with replying & editing as one function, but could maybe have them separate?

Reply:
I like it this way because they are mutually exclusive. this way we don't have to clear them individually, etc